docs/Apollo_Fuel/examples/sensor_calibration/lidar_to_gnss/lidar_to_gnss.config

```diff
@@ -6,7 +6,7 @@
 io_config: {
 	# task name now only support "camera_to_lidar, lidar_to_gnss"
 	task_name: "lidar_to_gnss"
-	output_path: "/apollo/sensor_calibration/lidar_to_gnss/extracted_data"
+	output_path: "/apollo/output/sensor_calibration/lidar_to_gnss/extracted_data"
 	# start_timestamp: "FLOAT_MIN"
```

Comment:
There is already a `/apollo/data` directory for storing output data. Besides, `/apollo/output` directory was already taken by `./apollo.sh release`.